diff --git a/content_access.module b/content_access.module
index ffb7681..60ea11e 100644
--- a/content_access.module
+++ b/content_access.module
@@ -684,8 +684,8 @@ function _content_access_get_node_permissions($type) {
 function content_access_get_role_gid($role) {
   $config = \Drupal::configFactory()->getEditable('content_access.settings');
   $roles_gids = $config->get('content_access_roles_gids');
-
-  return $roles_gids[$role];
+  return array_key_exists($role,$roles_gids) ? $roles_gids[$role] : "";
+  //return $roles_gids[$role];
 }

 /**
